<launch>
  <arg name="tank_id" default="0" />
  <node pkg ="tank_sdk" name ="acc_test_node" type ="acc_test_node" output = "screen">
    <remap from="~cmd" to="/tank_0/cmd"/>
    <param name="v0" value="1.5"/>
    <param name="v1" value="0.0"/>
    <param name="dur0" value="5.5"/>
    <param name="dur1" value="1.0"/>
  </node>
</launch>
